server/index.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const db = require('../database/index.js')
+const transactions = require('../database/Transaction.js')
+const stocks = require('../database/Stock.js')
+const port = 3333;
+
+// app.use(express.static('public'));
+app.use(bodyParser.json());
+
+app.get('/:stockTicker', (req, res) => {
+    console.log('PARAMMMMSSSSSS', req.params);
+    stocks.getStock(req.params.stockTicker, (err, data) => {
+        if (err) {
+            res.json({message: 'what\'s a stock?'});
+        } else {
+            res.json(data)
+        }
+    });
+});
+
+app.post('/', (req, res) => {
```

Comment:
probably shouldn't make POST requests to `/`, esp when GET reqs to `/` are supposed to serve the `index.html`

Reply:
fixed